### PR TITLE
Fixing the is_dir() calls.

### PR DESCRIPTION
**Pull request by burzum**
Fixing https://github.com/henrikbjorn/Lurker/issues/15

### CODE REVIEW

**Comment by henrikbjorn** [comment]
@burzum would you mind looking at the failing tests?

File: src/Lurker/ResourceWatcher.php [hunk 1]
```diff
@@ -89,7 +89,7 @@ public function track($trackingId, $resource, $eventsMask = FilesystemEvent::ALL
         if (!$resource instanceof ResourceInterface) {
             if (is_file($resource)) {
                 $resource = new FileResource($resource);
-            } elseif (is_dir($resource)) {
+            } elseif (is_dir($resource->getResource())) {
```
Comment (henrikbjorn) [inline src/Lurker/ResourceWatcher.php]: $resource isnt a ResourceInterface here but a string pointing to a directory.

**Comment by burzum** [comment]
I have no idea why the remaining tests fail, but I get a ton of them failing as well locally for the commit d13372695bf7853edc104bd6ee2ac3deaa92acc2 before my change.

**Comment by henrikbjorn** [comment]
Seems like the test was correct before you branch, would you mind looking in to it?

**Comment by burzum** [comment]
I might come back to the tests when I have enough time to fix 3rd party libs, which might take a few weeks up to month.

**Pull request closed without merging**